Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -0,0 +1,72 @@
+---
+versionFrom: 8.0.0
+---
+
+## Non-synchronised/replicated file system
+
+If the file system on your servers isn't performing any file replication then no _Umbraco_ configuration file changes are necessary.
+
+Depending on the configuration and performance of the local storage you might need to consider Examine Directory Factory Options and the Umbraco temporary storage location. 
+
+## Synchronised/replicated or shared file system
+
+If the file system on your servers is performing file replication then the Umbraco temporary folder must be excluded from replication.
+
+If the file system on your servers is located on shared storage you will need to configure Umbraco to locate the Umbraco temporary folder outside of the shared storage.
+
+### Replication techniques
+
+A common way to replicate files on Windows Server is to use [DFS](https://msdn.microsoft.com/en-us/library/windows/desktop/bb540031(v=vs.85), which is included with Windows Server.
+
+Additional DFS resources:
+
+* [Implementing DFS Replication in Windows Server 2003](http://www.windowsnetworking.com/articles_tutorials/Implementing-DFS-Replication.html)
+* [Overview of DFS Replication in Windows Server 2008 R2](https://technet.microsoft.com/en-us/library/cc771058.aspx)
+* [Watch an intro to installing and working with DFS](https://www.youtube.com/watch?v=DYfBoUt2RVE)
+
+There are other alternatives for file replication out there, some free and some licensed. You'll need to decide which solution is best for your environment.
+
+### Non-replicated files
+
+When deploying Umbraco in a load balanced scenario using file replication, it is important to ensure that not all files are replicated - otherwise you will experience file locking issues. Here are the folders and files that should not be replicated:
+
+* ~/App_Data/TEMP/*
+	* Alternatively store the Umbraco temporary files in the local server's 'temp' folder and set Examine to use a [Directory Factory](#examine-directory-factory-options). Achieve this by changing this configuration setting to 'true' in the web.config. The downside is that if you need to view this configuration file you'll have to find it in the temp files. Locating the file this way isn't always clear.
+			
+    ```xml
+    <add key="Umbraco.Core.LocalTempStorage" value="EnvironmentTemp" />
+    ```
+            
+* ~/App_Data/Logs/*
+	* This is **optional** and depends on how you want your logs configured (see below) 
+
+If for some reason your file replication solution doesn't allow you to not replicate specific files folders (which it should!!) then you can use an alternative approach by using virtual directories. *This is not the recommended setup but it is a viable alternative:*
```

Comment:
Think this should have a heading to split it up from the good practice above, it's more an aside for a niche situation...